libs/search-widget/src/common/expander/Expander.svelte

```diff
@@ -33,7 +34,11 @@
 </script>
 
 <div class="sw-expander">
-  <slot name="header" />
+  <div
+    on:click={() => (expanded = !expanded)}
+    class="header">
+    <Icon name={expanded ? 'chevron-down' : 'chevron-right'} /><slot name="header" />
```

Comment:
To be able to have a nice transition when toggling the expander, it would be better to use only one chevron (let say `chevron-right`) and to add a class on the div when expanded which would transform the icon (like `transform: rotate(90deg)`). Don't forget to add the transition (something like `transition: transform var(--transition-fast)`). 
(I wrote all that from memory, so double check the variable name, but the css variable for transition exists already)